yarn-project/end-to-end/src/utils.ts

```diff
@@ -62,30 +62,26 @@ type TxContext = {
 };
 
 /**
- * Sets up the environment for the end-to-end tests.
+ * Sets up the environment for the end-to-end tests without deploying contracts.
  * @param numberOfAccounts - The number of new accounts to be created once the RPC server is initiated.
+ * @param aztecNode - Instance of AztecNode implementation.
+ * @param firstPrivKey - The private key of the first account to be created.
+ * @param logger - The logger to be used.
  */
-export async function setup(numberOfAccounts = 1): Promise<{
-  /**
-   * The Aztec Node service.
-   */
-  aztecNode: AztecNodeService;
+export async function setupWithoutDeployingContractsAndAztecNode(
```

Comment:
Maybe `setupAztecRPCServer` would be a nicer name? I'm fine if you prefer to keep this one though.

Reply:
Addressed in [aeac0ae](https://github.com/AztecProtocol/aztec-packages/pull/1103/commits/aeac0aec9e883238be4cd91a2b7380c52aef870b)